lib/libzpool/zrepl_mgmt.c

```diff
@@ -545,10 +545,10 @@ uzfs_zvol_destroy_snapshot_clone(zvol_state_t *zv, zvol_state_t *snap_zv,
 	    clone_zv->zv_name, clonename, zv->zv_name);
```

Comment:
here as well, we are using clone_zv and snap_zv.. so, these two can't be NULL

Reply:
yes. In this function, there are no checks regarding non-null of clone and snap zv. Error checking should be added to this function. I will update this function with error checking of all zv.